micro-frontends/src/ipd/index.js

```diff
@@ -0,0 +1,46 @@
+// Adapt all IPD components to use within angular
+"use strict";
+
+import { react2angular } from "react2angular";
+import { IpdDashboard } from "./IpdDashboard";
+import { DrugChartModal } from "./DrugChartModal";
+import "bahmni-carbon-ui/styles.css";
```

Comment:
@AshishkurianTw Could you kindly confirm if the addition of styles.css is relevant to this PR? It was not included in the previous PR, so I'm curious to understand if its inclusion was deliberate and the reasoning behind it.